package.json

```diff
@@ -8,42 +8,47 @@
   },
   "scripts": {
     "lint:tests": "eslint --ext .js --fix tests/",
+    "ci": "npx codeceptjs run-multiple parallel --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@dbaas)^(?!.*@ami-upgrade)^(?!.*@pmm-upgrade)^(?!.*@qan)^(?!.*@nightly)'",
     "e2e": "npx codeceptjs run-multiple parallel --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
     "e2e:ia": "npx codeceptjs run -c pr.codecept.js --grep '@ia'",
     "e2e:dbaas": "npx codeceptjs run -c pr.codecept.js --grep '@dbaas'",
     "e2e:grafana-pr": "npx codeceptjs run -c pr.codecept.js --grep '@grafana-pr'"
   },
   "dependencies": {
-    "@types/lodash": "^4.14.168",
-    "@types/mocha": "^8.2.2",
-    "axios": "^0.21.4",
-    "codeceptjs": "^3.0.6",
-    "codeceptjs-dbhelper": "^1.0.1",
+    "@codeceptjs/configure": "^0.7.0",
+    "@codeceptjs/helper": "^1.0.2",
+    "@codeceptjs/ui": "^0.4.6",
+    "@types/lodash": "^4.14.177",
+    "@types/mocha": "^9.0.0",
+    "axios": "^0.24.0",
+    "codeceptjs": "^3.2.2",
+    "codeceptjs-dbhelper": "^1.2.2",
     "codeceptjs-mailosaurhelper": "^1.0.5",
     "database-js-mysql": "^1.1.3",
-    "eslint": "^7.24.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-prettier": "^8.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "dotenv": "^10.0.0",
+    "eslint": "^8.3.0",
+    "eslint-config-airbnb": "^19.0.1",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-no-inline-styles": "^1.0.5",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.23.2",
-    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.27.1",
+    "eslint-plugin-react-hooks": "^4.3.0",
     "faker": "^5.5.3",
     "form-data": "^4.0.0",
-    "generate-password": "^1.6.0",
+    "generate-password": "^1.7.0",
     "lodash": "^4.17.21",
-    "mocha": "^7.2.0",
-    "mocha-junit-reporter": "^1.23.1",
-    "mocha-multi": "^1.1.3",
-    "mochawesome": "^6.2.2",
+    "mocha": "^9.1.3",
+    "mocha-junit-reporter": "^2.0.2",
+    "mocha-multi": "^1.1.5",
+    "mochawesome": "^7.0.1",
     "mochawesome-merge": "^4.2.0",
-    "mochawesome-report-generator": "^5.2.0",
+    "mochawesome-report-generator": "^6.0.1",
     "moment": "^2.29.1",
-    "mongodb": "^3.7.2",
-    "playwright": "^1.10.0",
-    "prettier": "^2.2.1",
+    "mongodb": "^4.2.0",
+    "playwright": "^1.16.3",
+    "prettier": "^2.4.1",
```

Comment:
updated all the deps. latest codecept has interesting features, like:
 - [trace](https://codecept.io/playwright/#trace)
 - [video recording](https://codecept.io/playwright/#video)
 - [mocking requests](https://codecept.io/playwright/#mocking-network-requests)